flow-typed/box-ui-elements.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
Why are ui-elements types needed?

Reply:
yup. I didn't want to put all the flow types into box-annotations.js